### PR TITLE
sqs: do not send 'boleto' to queue if issuer 'development'

### PR DESCRIPTION
**Pull request by marquinhoalm**
Do not send 'boleto' to queue if issuer 'development'

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel confortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/resources/boleto/index.ts [hunk 1]
```diff
@@ -49,7 +49,12 @@ export const create = (event, context, callback) => {
 
   // eslint-disable-next-line
   const pushBoletoToQueueConditionally = (boleto) => {
-    if (boleto.status === 'pending_registration') {
+    const shouldSendToQueue = (boleto) => and(
```
Comment (grvcoelho) [inline src/resources/boleto/index.ts]: Renomeia para `shouldSendBoletoToQueue`

Comment (marquinhoalm) [inline src/resources/boleto/index.ts]: Há necessidade de utilizamos tanto o nome boleto? Sendo que trabalha apenas com boleto?

File: src/resources/boleto/index.ts [hunk 2]
```diff
@@ -49,7 +49,12 @@ export const create = (event, context, callback) => {
 
   // eslint-disable-next-line
```
Comment (mccraveiro) [inline src/resources/boleto/index.ts]: Qual rule estamos desativando aqui e porque?

File: src/resources/boleto/index.ts [hunk 3]
```diff
@@ -49,7 +49,12 @@ export const create = (event, context, callback) => {
...
   const pushBoletoToQueueConditionally = (boleto) => {
-    if (boleto.status === 'pending_registration') {
+    const shouldSendBoletoToQueue = and(
+      equals('pending_registration', prop('status', boleto)),
```
Comment (mccraveiro) [inline src/resources/boleto/index.ts]: Acredito que de para simplificar assim:
```js
const propNotEq = complement(propEq)
const shouldSendToQueue = and(
  propEq('status', 'pending_registration'),
  propNotEq('issuer', 'development')
)

if (shouldSendToQueue(boleto)) {
```

File: src/resources/boleto/index.ts (hunk 2 above)
Comment (grvcoelho) [inline src/resources/boleto/index.ts]: Não sei :( vamos arrumar isso quando tivermos o tslint :)

**Pull request merged**